.github/workflows/Release.yml

```diff
@@ -0,0 +1,38 @@
+name: Publish a release
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - '*'
+
+jobs:
+  publish_artifacts:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout the repo
+        uses: actions/checkout@v2
+
+      - uses: actions/cache@v2.1.6
+        with:
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+
+      - uses: actions/setup-java@v2
+        with:
+          distribution: 'adopt'
+          java-version: 11
+
+      - name: Publish the artifacts
+        env:
+          ORG_GRADLE_PROJECT_mavenCentralRepositoryUsername: ${{ secrets.SONATYPE_NEXUS_USERNAME }}
+          ORG_GRADLE_PROJECT_mavenCentralRepositoryPassword: ${{ secrets.SONATYPE_NEXUS_PASSWORD }}
+          ORG_GRADLE_PROJECT_signingKey: ${{ secrets.ARTIFACT_SIGNING_PRIVATE_KEY }}
+        run: ./gradlew publishAllPublicationsToMavenRepository --no-daemon
+
+env:
+  GRADLE_OPTS: -Dorg.gradle.configureondemand=true -Dorg.gradle.parallel=false -Dkotlin.incremental=false -Dorg.gradle.jvmargs="-Xmx3g -XX:MaxPermSize=2048m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8"
```

Comment:
@JakeWharton could you review this github actions portion, i'm less savvy on these things?

@eygraber alternatively, can this change land without github actions?  given it's stacked, this will block review of the subsequent PRs 😞

Reply:
If `gradle-mvn-push.gradle` is removed in #417 then this could wait. Alternatively if someone could figure out how to fix `gradle-mvn-push.gradle` then this could wait as well.

Reply:
Confirmed via local build.  Let's move Github Actions setup to another PR to get this in?

Reply:
https://github.com/JakeWharton/timber/pull/425